### DISCUSSION TITLE
[Stages] Reflection group to implement yscale

### DISCUSSION THREAD

**Discussion by AshtonTurner** [ikemen-engine/Ikemen-GO#1360] [Feature Request] [answered]
### Is your feature request related to a problem? Please describe.

So mugen fumbled this simple idea given how shadows work. If you're trying to make a stage with a lake or something similar from a minor distance you may know that you cant really depict how water skews your reflection depending on how close or far away you are from it. I'm not asking for absolute precision, but I do feel using the yscale would help with that. I've provided examples of what that might look like with where the shadow is skewed vs how reflection just draws the sprites inverted with no control.

fig1: what reflection currently looks like
![ikemen093](https://github.com/ikemen-engine/Ikemen-GO/assets/85361890/0ea4ea15-f8a4-46e7-abd2-7267f684d80c)
fig2: what shadows when skewed look like
![ikemen094](https://github.com/ikemen-engine/Ikemen-GO/assets/85361890/eea09a6f-f0ee-4744-ac5b-d2acb2f83a91)
both for visual difference.
![ikemen095](https://github.com/ikemen-engine/Ikemen-GO/assets/85361890/eb9e9982-f328-4815-b0b3-837dcf7b5a28)

Ultimately I think it will lend to more creative stages and further exemplify how useful this engine can be. 

### Describe the solution you'd like

In the [Reflection] group, the yscale parameter works the exact same way it does under [Shadow]. I imagine the code couldn't be too complicated to replicate it since we're just saying to do it for reflection group

### Search terms

reflection shadows stage-enhancements

**Comment by K4thos** [comment, marked as answer]
I think this is already implemented in nightly since [Shadow] and [Reflaction] groups now share the same parameters?
